packages/zwave-js/src/lib/node/Node.ts

```diff
@@ -3460,7 +3513,9 @@ protocol version:      ${this.protocolVersion}`;
 				// We don't know what this notification refers to, so we don't force a reset
 				allowIdleReset = false;
 			} else if (valueConfig.type === "state") {
-				allowIdleReset = valueConfig.idle;
+				// CL:0071.01.52.07.2: A controlling node SHOULD NOT use timeouts to consider a state variable
+				// to be idle [...] for v8 or newer supporting nodes.
+				allowIdleReset = command.version >= 8 && valueConfig.idle;
```

Comment:
According to the comment, shouldn't it be `command.version < 8`, only allowing timeout reset when it is v1-v7? You might also want to change the comment where this variable is used:
`// Nodes before V8 (and some misbehaving V8 ones) don't necessarily reset the notification to idle.`